### ISSUE TITLE
Porting interface to Lightning Design System

### ISSUE THREAD

**Issue by b-n** [SFDO-Community/declarative-lookup-rollup-summaries#487]
Love the tool, saves me from writing a lot of repeated code. I owe you many beverages.

Just wondering whether you had planned to update the interface to a lightning design system and/or lightning components? If not, I might be able to pick this up and do a lot of the grunt work. Preference would be to build it using lightning components, however they can be exposed on visualforce using lightning out for people still using classic.

Just thought I'd ask before forking and spending a lot of time on this. Would be keen to hear your thoughts on whether you think it's needed, what functionality you would like to see, etc.

**Comment by wes1278** [comment]
@nzchicken Updating the UI has been on the radar for sure. I suppose it's just we all need time/help doing so. In the 1.x version of DLRS, I created an "enhanced" editor that never really made it fully into the wild. I loved it, but I wrote it and am bias. :)

this github issue holds the current task of porting the "better ui" to the 2.x version of DLRS (which uses custom metadata instead of a custom object) https://github.com/afawcett/declarative-lookup-rollup-summaries/issues/272

Feel free to strike up a conversation about what you would see as MVP for a new ui in Lightning. If we built it "right" we could replace the editor for everyone and be exposed as a lightning component or inside a VF page for an older school tab.
